### PR TITLE
[UI] Pair Chat Screen & Pairing Screen

### PR DESCRIPTION
**Pull request by AyaNady17**
## Description

Implemented 

- Pairing Screen 
- Pair Chat Screen
image
image

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (does not change functionality, e.g. code style improvements, linting)
- [ ] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Please include screenshots below if applicable.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] Tag the PR with the appropriate labels

### CODE REVIEW

**Comment by AyaNady17** [comment]
please review @Aarush-Acharya  @vrundraval24

**Review by chandansgowda — COMMENTED** [review]
please attach demo screenshots

**Comment by AyaNady17** [comment]
> please attach demo screenshots

done @chandansgowda

File: lib/views/new_widgets/new_room_app_bar.dart [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            FontAwesomeIcons.comments,
+          ),
+          onPressed: () {
+            Get.to(NewRoomChatScreen());
+          },
+        ),
+        Padding(
+          padding: const EdgeInsets.only(right: 16.0, left: 16),
+          child: GestureDetector(
+            onTap: () => Get.to(const NewProfileScreen()),
```
[the remark this answers is not in the document]
Comment (AyaNady17) [reply lib/views/new_widgets/new_room_app_bar.dart]: done

File: lib/routes/app_pages.dart [hunk 2]
```diff
@@ -206,12 +208,19 @@ class AppPages {
       page: () => const UserAccountScreen(),
     ),
     GetPage(
+      name: AppRoutes.newPairChatScreen,
+      page: () => NewPairChatScreen(),
+    ),
+    GetPage(
+      name: AppRoutes.newPairingScreen,
+      page: () => NewPairingScreen(),
```
[the remark this answers is not in the document]
Comment (AyaNady17) [reply lib/routes/app_pages.dart:216]: done

**Comment by Aarush-Acharya** [comment]
Hey @chandansgowda, this PR is good to merge from my side

**Comment by AyaNady17** [comment]
@chandansgowda  please check that

**Pull request merged**